tracardi/process_engine/action/v1/connectors/twitter/tweet/plugin.py

```diff
@@ -0,0 +1,114 @@
+from tracardi.domain.resources.twitter_resrouce import TwitterResourceCredentials
+from tracardi.service.plugin.domain.register import Plugin, Spec, Form, FormGroup, FormComponent, FormField, MetaData, \
+    Documentation, PortDoc
+from tracardi.service.plugin.domain.result import Result
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.service.storage.driver import storage
+import tweepy
+from .model.config import Configuration
+
+
+def validate(config: dict):
+    return Configuration(**config)
+
+
+class TwitterTweetAction(ActionRunner):
+    credentials: TwitterResourceCredentials
+    config: Configuration
+
+    async def set_up(self, init):
+        config = validate(init)
+        resource = await storage.driver.resource.load(config.source.id)
+
+        self.config = config
+        self.credentials = resource.credentials.get_credentials(self, output=TwitterResourceCredentials)
+
+    async def run(self, payload: dict, in_edge=None) -> Result:
+
+        authorization = tweepy.OAuthHandler(
+            self.credentials.consumer_key,
+            self.credentials.consumer_secret
+        )
+
+        authorization.set_access_token(
+            self.credentials.access_token,
+            self.credentials.access_token_secret
+        )
+
+        tweet_api = tweepy.API(authorization)
+        tweet = self.config.tweet
```

Comment:
It could be good to use a template for tweet this way you will be able to embed data from profile, event ,etc. 

add:

```python
dot = self._get_dot_accessor(payload)
template = DotTemplate()
tweet = template.render(self.config.tweet, dot)
```

and send tweet.

You can read about dot templates here http://docs.tracardi.com/notations/templates/